osf/models/user.py

```diff
@@ -1746,7 +1746,7 @@ def from_cookie(cls, cookie, secret=None):
         secret = secret or settings.SECRET_KEY
 
         try:
-            token = itsdangerous.Signer(secret).unsign(cookie)
+            token = itsdangerous.Signer(secret).unsign(cookie).decode().split('.')[0]
```

Comment:
And forgot to mention `.split('.')[0]` doesn't make sense here since here "token" is the "cookie value". It may work locally or on CI simply because cookie val doesn't have `.` in it.